.github/workflows/release-notification.yml

```
@@ -59,22 +59,31 @@ jobs:
           grep -iE 'BUG:|MIJN-[0-9]+-BUG' release_notes.txt > bugs.txt || true
           grep -iE 'MIJN-[0-9]+($|[^-])' release_notes.txt > mijn.txt || true
 
-          echo "chores<<EOF" >> $GITHUB_OUTPUT
-          cat chores.txt >> $GITHUB_OUTPUT
-          echo "EOF" >> $GITHUB_OUTPUT
+      - name: Output Categorized Notes
+        run: |
+          if [ -s chores.txt ]; then
+            echo "chores<<EOF" >> $GITHUB_OUTPUT
+            cat chores.txt >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
 
-          echo "features<<EOF" >> $GITHUB_OUTPUT
-          cat features.txt >> $GITHUB_OUTPUT
-          echo "EOF" >> $GITHUB_OUTPUT
+          if [ -s features.txt ]; then
+            echo "features<<EOF" >> $GITHUB_OUTPUT
+            cat features.txt >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
 
-          echo "bugs<<EOF" >> $GITHUB_OUTPUT
-          cat bugs.txt >> $GITHUB_OUTPUT
-          echo "EOF" >> $GITHUB_OUTPUT
+          if [ -s bugs.txt ]; then
+            echo "bugs<<EOF" >> $GITHUB_OUTPUT
+            cat bugs.txt >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
 
-          echo "mijn<<EOF" >> $GITHUB_OUTPUT
-          cat mijn.txt >> $GITHUB_OUTPUT
-          echo "EOF" >> $GITHUB_OUTPUT
-        shell: bash
+          if [ -s mijn.txt ]; then
+            echo "mijn<<EOF" >> $GITHUB_OUTPUT
+            cat mijn.txt >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
```

Cool dit lost het probleem op van al die headers zonder paragrafen.
Dit draait gewoon als een bash script toch? Ik zou hier een functie makkelijker begrijpen dus misschien dit:

```suggestion
        append_to_github_output() {
          if [ -s "$1.txt" ]; then
            echo "$1<<EOF" >> "$GITHUB_OUTPUT"
            cat "$1.txt" >> "$GITHUB_OUTPUT"
            echo "EOF" >> "$GITHUB_OUTPUT"
          fi
        }

        append_to_github_output 'chores'
        append_to_github_output 'features'
        append_to_github_output 'bugs'
        append_to_github_output 'mijn'
```